vpu/generic/vpuGeneric/.conf/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----

```

Comment:
@andreriesco is copying the same key pair to `.conf` for each sample really necessary?
Why are we not using a default insecure ssh connection and a warning banner via sshd_config?

Reply:
Tagging @microhobby also, maybe he has more details.

Reply:
Open https://github.com/toradex/torizon-samples/issues/16